src/FFMpeg/Media/Concat.php

```diff
@@ -91,7 +91,12 @@ public function saveFromSameCodecs($outputPathfile, $streamCopy = TRUE)
         $sourcesFile = $fs->createTemporaryFile('ffmpeg-concat');
 
         // Set the content of this file
-        $fileStream = fopen($sourcesFile, 'w') or throw new ExecutionFailureException('Cannot open the temporary file.');
+        try {
+            $fileStream = fopen($sourcesFile, 'w');
+        } catch (Exception $e) {
```

Comment:
That won't work, because `fopen` returns a `resource` or `false`.

Do it like this:
````php
$fileStream = fopen($sourcesFile, 'w');

if($fileStream === false) {
    throw new ExecutionFailureException('Cannot open the temporary file.');
}